postmark/assets/logs/postmark.yaml

```diff
@@ -0,0 +1,328 @@
+id: postmark
+metric_id: postmark
+backend_only: false
+facets:
+  - groups:
+      - Event
+    name: Event Name
+    path: evt.name
+    source: log
+  - groups:
```

Comment:
I see you are already mapping this below, is this intended?

```
  - groups:
      - Web Access
    name: URL Path
    path: http.url_details.path
    source: log
```

Reply:
No, that's not intended

Reply:
👍 I'll let you remove the extra one and after that I'll start testing the logs assets on our end!

Reply:
@brunorenier
I think this mapping was suggested when CI build was executed and from that we have added into our pipelines.
For reference, the current CI build is failing and it is suggesting us to add the same thing again.